install/install.go

```diff
@@ -1688,9 +1688,7 @@ func (k *K8sInstaller) Install(ctx context.Context) error {
 
 		s, err := collector.Status(ctx)
 		if err != nil {
-			if s != nil {
-				fmt.Println(s.Format())
-			}
+			fmt.Println(s.Format())
```

Comment:
This will now print an empty line instead of nothing at all. Is that intended?

Reply:
Well spotted! That is not intended. Will change to `fmt.Print` and add an additional newline to `s.Format()`s return value.